toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -257,6 +263,13 @@ private fun MapViewEventHandler(
                 }
             }
         }
+        launch(Dispatchers.Main.immediate) {
```

Comment:
`Dispatchers.Main.immediate` is only required for the gesture events. Events raised by Core will always be raised on a Core thread, so we cannot avoid a re-dispatch when collecting on the composition's coroutine context.
```suggestion
        launch() {
```

Same above fore `viewpointChanged`...